### PR TITLE
Add alignment options

### PR DESCRIPTION
**Pull request by danalvrz**
- Added schema enhancer with alignment options

### CODE REVIEW

**Review by sneridagh — APPROVED** [review]

**Comment by davisagli** [comment]
Can we move the styles into https://github.com/kitconcept/volto-separator-block/blob/main/src/theme/main.less as well? (Note, this will require converting from scss to less and using the less breakpoint variables)

File: src/theme/main.less [hunk 1]
```diff
@@ -1,20 +1,62 @@
-// Separator Block
+// //Separator Block
 .block.separator {
+  top: unset;
   display: block;
   height: 54px;
```
Comment (davisagli) [inline src/theme/main.less:5]: I think we should unset the height here (to override the one inherited from somewhere else) and let the height be derived automatically from the content and padding.

File: src/theme/main.less [hunk 2]
```diff
@@ -1,20 +1,62 @@
...
+  margin: 50px 0;
+  clear: both;
+
+  &:not(.inner):not([role='presentation']) {
+    padding: 25px 0 !important;
+  }
+
+  &::after {
+    display: none !important;
+  }
```
Comment (davisagli) [inline src/theme/main.less:16]: I think this can go. The block has no `after` pseudoelement.

File: src/theme/main.less [hunk 3]
```diff
@@ -1,20 +1,62 @@
...
 
   .line {
-    height: 54px;
+    width: unset;
+    height: unset;
```
Comment (davisagli) [inline src/theme/main.less:20]: I don't think these unsets are needed. When the styles were in the theme, it was unsetting the height that was set here. If we don't set a height here, it doesn't need to be unset.

File: src/theme/main.less [hunk 4]
```diff
@@ -1,20 +1,62 @@
...
 
-    &::after {
+    &:not(.inner)::after {
```
Comment (davisagli) [inline src/theme/main.less:22]: The `.line` element never has a `inner` class, so this doesn't do anything.
```suggestion
    &::after {
```

**Review by sneridagh — APPROVED** [review]
@davisagli LGTM, tell me if we can merge already after the last changes.

**Comment by davisagli** [comment]
@sneridagh yes, ready for merge and release as far as I am concerned.DavidOn Jan 16, 2023, at 1:30 AM, Víctor Fernández de Alba ***@***.***> wrote:﻿
@sneridagh approved this pull request.

@davisagli LGTM, tell me if we can merge already after the last changes.

**Pull request merged**